crates/biome_cli/src/cli_options.rs

```diff
@@ -18,7 +18,7 @@ pub struct CliOptions {
     #[bpaf(long("use-server"), switch, fallback(false))]
     pub use_server: bool,
 
-    /// Print additional diagnostics, and some diagnostics show more information.
+    /// Print additional diagnostics, and some diagnostics show more information. If any lints applied fixes, this flag will also print which files were fixed.
```

Comment:
This is true for any file that's modified, not just linting e.g. formatting and import sorting 

Can you update it?

Reply:
done